cypress/integration/articleSpec.js

```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
       .should('have.attr', 'src')
       .and('match', /(\/static\/js\/bundle\.\w+\.js)/g);
   });
+
+  it('should check the font file loads', () => {
+    const fontFamiliesArray = [];
+    cy.get('*')
+      .each(element => {
+        const fontFamily = Cypress.$(element).css('font-family');
```

Comment:
Do we have to do `Cypress.$(element)`, can we just do `element.css('font-family')`?